README.md

```diff
@@ -5,98 +5,119 @@
 
 [![slack](https://apiblueprint-slack.herokuapp.com/badge.svg)](http://apiblueprint-slack.herokuapp.com/)
 
-API Blueprint is a documentation-oriented API description language. A couple of semantic assumptions over the plain [Markdown](http://daringfireball.net/projects/markdown/).
+A powerful high-level API design language for web APIs.
 
-API Blueprint is perfect for designing your Web API and its comprehensive documentation but also for quick prototyping and collaboration. It is easy to learn and even easier to read – after all it is just a form of plain text.
+API Blueprint is simple and accessible to everybody involved in the API design
+lifecycle. Its syntax is concise yet expressive.
 
-API Blueprint, its parser, and most of its tools are completely open sourced so you don't have to worry about vendor lock-in. This also means you can freely integrate API Blueprint into any type of product, commercial or not.
+With API Blueprint you can quickly prototype and model APIs to be created or
+describe already deployed mission-critical APIs. From a [car][tesla] to the
+largest CDN in the world.
 
-## TL;DR
-+ Web API description language
-+ Pure Markdown
-+ Designed for humans
-+ Understandable by machines
-+ Open & free
+The API Blueprint is built to encourage dialog and collaboration between project
```

Comment:
Should this be `dialogue` not `dialog`, or is this equivalent in :us: English? I though dialog was one of these:

![iu-1](https://cloud.githubusercontent.com/assets/44164/11780283/b8a4cff0-a258-11e5-859c-dd6d9a7dd0ac.png)